integration_tests/serverless.yml

```diff
@@ -4,9 +4,6 @@ service: integration-tests-js
 provider:
   name: aws
   region: sa-east-1
-  tracing:
```

Comment:
I disabled X-Ray tracing in the integration tests. Due to a bug on the AWS side, the `XRAY` log lines are now improperly formatted.

Reply:
Is this the same issue we were seeing with Ruby?

Reply:
Sounds like it